ui/src/components/ExportDialog.tsx

```diff
@@ -78,7 +79,10 @@ export default function ExportDialog({ open, onClose }: Props) {
   };
 
   const handleExport = () => {
-
+    track({
+      action: "ExportVolume",
```

Comment:
Should we also track what radio button option was selected? i.e. local file, local image, new image, or registry.

Reply:
done, same for import. 

```
{
    "event": "eventVolumeBackupExtension",
    "properties": {
        "action": "ExportVolume",
        "app major version": 4,
        "app minor version": 12,
        "app patch version": 0,
        "app version name": "4.12.0",
        "channel": "main",
        "engine": "linux",
        "exportType": "toLocalFile",
        "os": "macos",
        "os language": "en",
        "os major version": "12",
        "os minor version": "4",
        "os patch version": "0",
        "sessionId": "684aa77c-9b77-4a95-bf92-507b9566947f",
        "volumeSize": 16000
    },
    "userId": "85D100AD-1E8F-4F78-9528-798B99DD41D7"
}
```